README.md

```diff
@@ -1,21 +1,91 @@
+master branch.**
+
+### UAC QID and Exception Manager Schemas
+
+The UAC QID and Exception Manager services have their own schemas, independent of the common entities schema defined in
+this repository. This repo also handles generating these schema definitions by pulling the services code and using it's
+hibernate models.
+
+If you have made changes to Exception Manager or the UAC-QID Service and you need to re-generate the ground zero DDL,
+you can specify specific branches with the following options:
 
-If you have made changes to Exception Manager or the UAC-QID Service and you need to re-generate the ground zero DDL, you can specify specific branches with the following options:
 - `UAC_QID_SERVICE_BRANCH`
 - `EXCEPTION_MANAGER_BRANCH`
 
-For example, you might want to run `UAC_QID_SERVICE_BRANCH=example-branch make dev-build` to get the groundzero DDL to be generated for an as-yet unmerged branch called `example-branch`.
+For example, you might want to run `UAC_QID_SERVICE_BRANCH=example-branch make dev-build` to get the groundzero DDL to
+be generated for an as-yet unmerged branch called `example-branch`.
 
-Once you have run `make dev-build` you will be able to see any differences between the old and the new DDL in Git, and create patch script(s) accordingly.
+Once you have run `make dev-build` you will be able to see any differences between the old and the new DDL in Git, and
+create patch script(s) accordingly.
 
 ## Patching a database
-The script `patch_database.py` is used by RM to run database patches from a tagged release of this repository. This script is invoked from our pipelines and will run in a Kubernetes pod to apply any database patches from files in the tagged release version of this repository.
 
-## Releasing this repo
-When tagging a release of this repo you must update the version and and patch number in [ddl_version.sql](groundzero_ddl/ddl_version.sql) and update the current_version variable in [patch_database.py](patch_database.py)
+The script `patch_database.py` is used by RM to run database patches from a tagged release of this repository. This
+script is invoked from our pipelines and will run in a Kubernetes pod to apply any database patches from files in the
+tagged release version of this repository.
+
+The database schema version number and patch numbers must then be updated in [patch_database.py](patch_database.py)
+and [ddl_version.sql](groundzero_ddl/ddl_version.sql), so that the patch script is aware of the current schema version.
+
+### Testing patch scripts
+
+The patch scripts can be tested by running our dev-common-postgres docker image from the latest main branch build, then
+running the database patches against it.
+
+If you have rebuilt the dev-common-postgres image locally it will already be on the updated schema version, and to test
+any local patches you must first pull the main branch build with:
+
+```shell
+make pull-latest-dev-common-postgres
```

Comment:
should this be: 
pull-latest-dev-postgres

or should it be renamed in the Makefil